core/jazz_ui/src/app/pages/service-overview/service-overview.component.ts

```diff
@@ -289,71 +318,222 @@ export class ServiceOverviewComponent implements OnInit {
+              this.isLoadingService = true;
+              this.serviceDetail.onDataFetched(Response.data.updatedService);
+              this.isLoadingService = false;
+              this.loadPlaceholders()
+              this.disp_show=true;
+              this.saveClicked = false;
+              let successMessage = this.toastmessage.successMessage(Response,"updateService");
+              this.toast_pop('success',"", "Data for service: "+this.service.name +" "+successMessage);
+            },
+            (Error)=>{
+                this.isLoadingService=false;
+                this.isPUTLoading = false;
+                this.disp_show=true;
+                this.saveClicked = false;
+                this.edit_save='SAVE';
+                let errorMessage = this.toastmessage.errorMessage(Error,"updateService");
+                this.toast_pop('error', 'Oops!', errorMessage)
+                // this.toast_pop('error','Oops!', "Data cannot be updated. Service Error.");
             });
-            this.service.email = email_temporary;
-            this.service.slackChannel = slack_temporary;
-
-            this.isLoadingService = false;
-            this.disp_edit = true;
-            this.showCancel = false;
-            this.disp_show = false;
-            this.edit_save = 'EDIT';
-            let successMessage = this.toastmessage.successMessage(Response, "updateService");
-            this.toast_pop('success', "", "Data for service: " + this.service.name + " " + successMessage);
-            this.check_empty_fields();
-          },
-          (Error) => {
-            this.isLoadingService = false;
-            this.disp_edit = false;
-            this.disp_show = true;
-            this.edit_save = 'SAVE';
-            let errorMessage = this.toastmessage.errorMessage(Error, "updateService");
-            this.toast_pop('error', 'Oops!', errorMessage)
-          });
+
+
+  }
+  onAdvancedSaveClick(){
+    this.saveClicked = false;
+    this.advancedSaveClicked = true;
+    let payload = {};
+
+    if( this.advancedSaveClicked){
+      if (this.rateExpression.type != 'none') {
+        this.rateExpression.cronStr = this.cronParserService.getCronExpression(this.cronObj);
+        if (this.rateExpression.cronStr == 'invalid') {
+          return;
+        } else if (this.rateExpression.cronStr !== undefined) {
+          payload["rateExpression"] = this.rateExpression.cronStr;
+        }
+      }
+
+      if (this.eventExpression.type !== "awsEventsNone") {
+        var event = {};
+        event["type"] = this.eventExpression.type;
+        if (this.eventExpression.type === "dynamodb") {
+          event["source"] = "arn:aws:dynamodb:us-west-2:302890901340:table/" + this.eventExpression.dynamoTable;
+          event["action"] = "PutItem";
+        } else if (this.eventExpression.type === "kinesis") {
+          event["source"] = "arn:aws:kinesis:us-west-2:302890901340:stream/" + this.eventExpression.streamARN;
```

Comment:
Remove the hardcoded account value.

Reply:
Fetching account number and region from config file in the new pr (PR 549)